pkg/apis/pxc/v1/pxc_types.go

```diff
@@ -868,6 +869,11 @@ func (cr *PerconaXtraDBCluster) CheckNSetDefaults(serverVersion *version.ServerV
 				strg.Volume.reconcileOpts()
 			}
 		}
+
+		if c.Backup.AllowParallel == nil {
+			b := false
```

Comment:
We need to allow it if we do not have any options in cr. We should not change the behavior of existing clusters. @spron-in  Am I right? Also please check the option name.

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/1413/commits/7e66ecc42c2998dc0d18602597aa8f9a72285d02